### PR TITLE
public s3 no copy release notes

### PR DESCRIPTION
**Pull request by essiequoi**
# What does this PR do?

Release notes for public s3 no copy

### CODE REVIEW

**Comment by essiequoi** [comment]
Still in progress, to add additional fixes to the list starting with ✔️

**Review by rlizzo — APPROVED** [review]

File: changelog/2022-04-19.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+In these cases, you can pass the `--no-copy` flag to the `grid datastore create` command. This flag will
+prevent grid from making a copy of the dataset, speeding up datastore creation time significantly when working with large buckets. 
+
+Example:
+`grid datastore create S3://ruff-public-sample-data/esRedditJson --no-copy` 
+
+:::note
+Please note that direct access to private S3 buckets is not currently supported.
+:::
```
Comment (essiequoi) [inline changelog/2022-04-19.md:23]: @RobertLaurella lines 7-23 are where I need your review. Thanks!

Comment (RobertLaurella) [inline changelog/2022-04-19.md:23]:
```suggestion
#### :star: Faster S3 Datastores!

We are happy to announce that, as of today, creating datastores from S3 buckets is almost instant!

In most cases, your S3 bucket will fit one (or both) of the following criteria:
1. the bucket is continually updating with new data which you want included in a Grid datastore 
2. the bucket is particularly large (leading to long datastore creation times)

In both of these cases, you can pass the `--no-copy` flag to the `grid datastore create` command. This flag will
prevent grid from making a copy of the dataset, which significantly speeds up datastore creation time when working with large buckets or ones that you want to continually update with new files.

Here's an example:
`grid datastore create S3://ruff-public-sample-data/esRedditJson --no-copy` 

:::note
Please note that direct access to private S3 buckets is not currently supported.
:::
```

**Review by RobertLaurella — CHANGES_REQUESTED** [review]

File: changelog/2022-04-19.md (hunk 1 above)
Comment (essiequoi) [reply changelog/2022-04-19.md:23]: This flag will prevent grid from making a copy of the dataset, which significantly speeds up datastore creation time when working with large buckets or when making incremental updates to your bucket that do not require the entire dataset to be uploaded again. @RobertLaurella

Comment (RobertLaurella) [reply changelog/2022-04-19.md:23]: Two options. Square brackets are optional:

This flag will prevent Grid from making a copy of the dataset, which significantly speeds up datastore creation time when working with large buckets or if you do not want to reupload the entire dataset every time [you add a new file].

This flag will prevent Grid from making a copy of the dataset, which significantly speeds up datastore creation time when working with large buckets or when you intend to make incremental changes to your bucket and do not want to reupload the entire dataset each time [you add a new file].

File: changelog/2022-04-19.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+#### :star: Faster S3 Datastores!
+
+We are happy to announce that, as of today, creating datastores from S3 buckets is almost instant!
+
+In most cases, your S3 bucket will fit one (or both) of the following criteria:
+1. the bucket is continually updating with new data which you want included in a Grid datastore 
+2. the bucket is particularly large (leading to long datastore creation times)
+
+In both of these cases, you can pass the `--no-copy` flag to the `grid datastore create` command. This flag will
+prevent grid from making a copy of the dataset, which significantly speeds up datastore creation time when working with large buckets or ones that you want to continually update with new files.
```
Comment (essiequoi) [inline changelog/2022-04-19.md:16]:
```suggestion
In both of these cases, you can pass the `--no-copy` flag to the `grid datastore create` command. This flag will prevent Grid from making a copy of the dataset, which significantly speeds up datastore creation time when working with large buckets or when you intend to make incremental changes to your bucket and do not want to re-upload the entire dataset each time you add a new file.
```

**Review by essiequoi — COMMENTED** [review]
Update to one sentence

**Comment by essiequoi** [comment]
@RobertLaurella ready for your final review. If good please approve :)

**Review by RobertLaurella — APPROVED** [review]

**Review by RobertLaurella — APPROVED** [review]

**Pull request merged**